hs_communities/views/communities.py

```diff
@@ -80,6 +80,7 @@ def get_context_data(self, *args, **kwargs):
                                                                       privilege=PrivilegeCodes.OWNER).exists() else 0
                 data["is_admin"] = is_admin
                 data["user"] = user_json(user)
+                context["is_admin"] = is_admin
```

Comment:
@pkdash I'm not seeing where we need this in the context but I bet you're right that we do...

Reply:
If you try to add an owner, without this change the add owner modal won't be displayed. I guess it's not related to timeout issue though.

Reply:
got it thanks